CONTRIBUTING.md

```diff
@@ -63,8 +63,25 @@ please [set one up](http://golang.org/doc/code.html).
 
 The version of GO should be **1.11** or above.
 
-After installation, you'll need `GOPATH` defined,
-and `PATH` modified to access your Go binaries.
+After installation, there are two ways to build TiDB binary.
+
+#### For a quick taste
+
+The `GOPATH` is not necessary.
+
+```
+mkdir tmp
```

Comment:
why not using `git clone`?

Reply:
```sh
git clone https://github.com/pingcap/tidb.git
cd tidb
make
```

Reply:
Makefile is convenient, but if you don't want it, `go get` is a good substitution, especially for gophers!

Reply:
Yeah. How about adding another section for non gophers about how to have a quick taste? Maybe they will fall in love with golang from this first beginning?

Reply:
@zz-jason 
Those steps are for non gophers:
https://github.com/pingcap/tidb/pull/8814/files#diff-6a3371457528722a734f3c51d9238c13R97

Reply:
1. It doesn't need clone the repo
2. It doesn't require GOPATH

That's a big deal comparing to `git clone+make`
You know, the single `make` command still need to get go module  dependency, so it's much heavier.
@zz-jason

Reply:
Okay